### ISSUE TITLE
Soup pots can be used as a poor man's chem goggle.... Sometimes?

### ISSUE THREAD

**Issue by AdipemDragon** [tgstation/tgstation#76984]
Reporting client version: 514.1589

## Round ID:
210643

## Reproduction:
![image](https://github.com/tgstation/tgstation/assets/76874615/275c9f9f-9ca2-4bad-b1db-4b4f9d907643)
Pick up a Who Knows What chemical with a beaker, watch as you can't see what it is in the soup pot.
Pick up Nutraslop with the beaker, it's still unknown, but putting it in the soup pot reveals what it is to the naked eye? 

Are all chemicals in the edible sphere, then, stuff that the soup pot reveals? (Will test more thoroughly later)

**Comment by MrMelbert** [comment]
Yes, this is intended, the Soup Pot reveals any `/consumable` subtype reagents to make it easier for chefs to see what they're cooking up. This was done so it's not a necessity that the chef go hunt down science goggles to actually see what they're cooking. Bowls reveal likewise.

I intended on changing this with "taste testing" the pot at some point to see all `/consumable` reagents within at some point but this was the temporary measure.
